src/main/java/net/neoforged/neoforge/common/NeoForgeMod.java

```diff
@@ -301,7 +303,8 @@ public void setItemMovement(ItemEntity entity) {
             .sound(SoundActions.BUCKET_FILL, SoundEvents.BUCKET_FILL)
             .sound(SoundActions.BUCKET_EMPTY, SoundEvents.BUCKET_EMPTY)
             .sound(SoundActions.FLUID_VAPORIZE, SoundEvents.FIRE_EXTINGUISH)
-            .canHydrate(true)) {
+            .canHydrate(true)
+            .addDripstoneDripping(0.17578125F, ParticleTypes.DRIPPING_DRIPSTONE_WATER, Blocks.WATER_CAULDRON, SoundEvents.POINTED_DRIPSTONE_DRIP_WATER_INTO_CAULDRON)) {
```

Comment:
Magic numbers make me itch. Based on [the Minecraft Wiki](https://minecraft.wiki/w/Pointed_Dripstone#Cauldrons), the percent chance for a cauldron being filled with water by pointed dripstone stalactites is 45/256, which (according to my calculators) equals to this number exactly.

Reply:
I opted to use the WATER_TRANSFER_PROBABILITY_PER_RANDOM_TICK in PointedDripstoneBlock for this instead so its less of a magic number. I'm currently patching it to be public, but if you would rather me use an AT for it, let me know.

Reply:
I didn't realize those constants existed, as I didn't look for them. 😄 Please change those to ATs, since we generally want to avoid modifying access through patches (in part because modders would be unable to modify via AT what we've modified via patches).